.github/workflows/gh-pages.yml

```diff
@@ -29,9 +29,12 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
 
-      - name: Install      
+      - name: Install
         run: npm install
 
+      - name: Install docsy dependencies
```

Comment:
This part is missing in : 
- .devcontainer/devcontainer.json "postCreateCommand" step
- .gitpod.yml "before" step
A way to mutualize this script in package.json would nice, so I'll try to update it in a next PR if you prefer